recipes-multimedia/alsa/alsa-lib/0001-pcm-dmix-Fix-resume-with-multiple-instances.patch

```diff
@@ -0,0 +1,64 @@
+From 2adc30e9838daf62312f32a0a93129d64b1668b3 Mon Sep 17 00:00:00 2001
```

Comment:
for this commit log I would use:
alsa-lib: Update to L6.6.52-2.2.0 release 

While at that, add a fix for resume with multiple instances and remove the overrides for the toolchain as they are not needed any more.

Reply:
Has this been validated against the master branch? I have the impression that this has been not. I believe this same fix is included in the newest version of the recipe and the build fails because the patch does not apply.

Reply:
I can confirm that the patch does not apply on master. It appears that the source now may contain the idea of the patch, though it is not exact.